sdk/containerregistry/Azure.Containers.ContainerRegistry/src/ContainerRegistryClient.cs

```diff
@@ -23,7 +23,7 @@ public partial class ContainerRegistryClient
         /// <summary>
         /// Initializes a new instance of the <see cref="ContainerRegistryClient"/> for managing container images and artifacts,
         /// using anonymous access to the registry.  Only operations that support anonymous access are enabled.  Other service
-        /// methods will throw <see cref="RequestFailedException"/> if called.
+        /// methods will throw <see cref="RequestFailedException"/> if called from this client.
```

Comment:
Is this also documented in an  `<exception` tag for each operation that exhibits this behavior?

Reply:
Technically yes, as I did a pass to add these whereever the method makes a service call.  The service is changing which endpoints allow anonymous access, so we decided not to indicate it method-by-method.